src/hotspot/share/runtime/vmThread.hpp

```diff
@@ -123,6 +124,9 @@ class VMThread: public NamedThread {
   static PerfCounter* perf_accumulated_vm_operation_time() {
     return _perf_accumulated_vm_operation_time;
   }
+  static PerfVariable* perf_vm_thread_cpu_time() {
+    return _perf_vm_thread_cpu_time;
+  }
```

Comment:
Why do you need an accessor for the cpu-time variable in the VMThread but none of the others?